app/ports/tests/test_dependencies.py

```diff
@@ -1,18 +1,17 @@
 import os
 
-from django.test import TestCase, Client
+from django.test import TransactionTestCase, Client
 from django.urls import reverse
 
 from ports.models import Dependency, Port
 from MacPorts.config import TEST_PORTINDEX_JSON
 
 
-class TestDependencies(TestCase):
+class TestDependencies(TransactionTestCase):
+    reset_sequences = True
+
     def setUp(self):
         self.client = Client()
```

Comment:
I doubt you need `self.client = Client()`. It is there in the base class `SimpleTestCase`.